server/views/templates/article.njk

```diff
@@ -1,33 +1,60 @@
 {% extends "pages/article.njk" %}
 
-{% block articleBody %}
-  <div class="body-content">
-    {% for bodyPart in bodyParts %}
-      <div class="grid grid--dividers grid--flush-dividers">
-      {% if bodyPart.copy %}
-        <div class="grid__cell grid__cell--xl6 grid__cell--shift-xl1 grid__cell--l7 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-          {{ bodyPart.copy | safe }}
-        </div>
-        {% if bodyPart.figure %}
-          <div class="grid__cell grid__cell--xl4 grid__cell--l5 grid__cell--m6 grid__cell--shift-m1 grid__cell--s4">
-            {% component 'figure', {
-              type: 'picture',
-              sources: bodyPart.figure.sources,
-              caption: bodyPart.figure.caption,
-              modifiers: ['sticky'],
-              icon: bodyPart.figure.icon } %}
+{% block body %}
```

Comment:
There'll need to be some refactoring here - rewriting the whole body block will make this really hard to use in the future if we're looking to it as a source of truth.